pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ServerCnxTest.java

```diff
@@ -2153,4 +2167,157 @@ public void testHandleAuthResponseWithoutClientVersion() {
         verify(authResponse, times(1)).hasClientVersion();
         verify(authResponse, times(0)).getClientVersion();
     }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    void shouldFailHandleLookup() throws Exception {
+        resetChannel();
+        Field stateUpdater = ServerCnx.class.getDeclaredField("state");
+        stateUpdater.setAccessible(true);
+        stateUpdater.set(serverCnx, ServerCnx.State.Failed);
+
+        CommandLookupTopic commandLookupTopic = spy(CommandLookupTopic.class);
+        serverCnx.handleLookup(commandLookupTopic);
```

Comment:
Maybe  `serverCnx.handleLookup(any(CommandLookupTopic.class))` is better :)

Reply:
Good